### PR TITLE
fixes #73 height issue

### LINKED ISSUE #73: Homepage height / designer height issue
On the homepage of the designer, there is a strange height issue happening (you can reproduce this on designer.grommet.io):

GrommetDesignerHeightBug

It seems to be related to the height: '100%' set on the App.js

If you instead remove that inline style, it works on the homepage but breaks the designer. But if you inline `style={{ height: '100vh' }}` in the Designer.js root Grid component, it works as expected:

GrommetDesignerHeightBugFixed

### PR DESCRIPTION
**Pull request by jjwilliams42**
This is in reference to #73, it should allow the homepage to work as expected as well as the designer to fill the screen.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -277,7 +277,6 @@ const App = () => {
       theme={designerTheme}
       themeMode={colorMode}
       dir={rtl ? 'rtl' : undefined}
-      style={{ height: '100%' }}
```
Comment (ericsoderberghp) [inline src/App.js]: I seem to recall this is needed for reasons I can't remember. :(

File: src/Designer.js [hunk 2]
```diff
@@ -264,7 +264,7 @@ const Designer = ({ colorMode, design, setDesign }) => {
 
   return (
     <Keyboard target="document" onKeyDown={onKey}>
-      <Grid fill columns={columns} rows="full">
+      <Grid fill columns={columns} rows="full" style={{ height: '100vh' }}>
```
Comment (ericsoderberghp) [inline src/Designer.js]: I think `fill` should be adding `height: 100%`, which should work, as long as App.js isn't changed.

**Review by ericsoderberghp — COMMENTED** [review]
Instead of this approach, I think we should change Start.js to use `fill`, `flex`, and `overflow` better. For example:
```
--- a/src/Start.js
+++ b/src/Start.js
@@ -149,8 +149,8 @@ const Start = ({
   }, [search]);
 
   return (
-    <Box direction="row-responsive" gap="medium" border="between">
-      <Box align="start" pad="large" height="100vh">
+    <Box fill direction="row-responsive" gap="medium" border="between">
+      <Box align="start" pad="large">
         <Heading margin={{ top: 'none' }}>grommet designer</Heading>
         <Paragraph size="xlarge">design with grommet components</Paragraph>
         <Button
@@ -230,9 +230,15 @@ const Start = ({
         </Box>
       </Box>
 
-      <Box flex pad={{ vertical: 'medium', horizontal: 'large' }} gap="large">
+      <Box
+        flex
+        fill
+        overflow="auto"
+        pad={{ vertical: 'medium', horizontal: 'large' }}
+        gap="large"
+      >
         {designs && designs.length > 0 && (
-          <Box>
+          <Box flex={false}>
             <Header>
               <Heading level={2}>my designs</Heading>
               {designs.length > 5 && (
@@ -287,7 +293,7 @@ const Start = ({
           </Box>
         )}
 
-        <Box alignSelf="start">
+        <Box flex={false} alignSelf="start">
           <Header alignSelf="stretch">
             <Heading level={2}>tutorials</Heading>
           </Header>
@@ -311,7 +317,7 @@ const Start = ({
           ))}
         </Box>
 
-        <Box>
+        <Box flex={false}>
           <Heading level={2}>inspiration</Heading>
           <Box direction="row" overflow="auto" gap="medium">
             {inspirations.map(({ title, url }) => (
@@ -323,7 +329,7 @@ const Start = ({
         </Box>
 
         {readme && (
-          <Box>
+          <Box flex={false}>
             <Markdown>{readme}</Markdown>
           </Box>
         )}
```

File: src/App.js (hunk 1 above)
Comment (jjwilliams42) [reply src/App.js]: I do like that approach better, I will give it a try

Comment (jjwilliams42) [reply src/App.js]: Hrmm, this change works everywhere well except the my designs area (it's all collapsed - I can't see my thumbnails). Looking more into it.

**Comment by jjwilliams42** [comment]
@ericsoderberghp I figured out the last issue with the "my designs" area being collapsed, everything should work as expected now.

**Review by ericsoderberghp — APPROVED** [review]

**Comment by ericsoderberghp** [comment]
Thanks for your contribution!

**Pull request merged**